R/plots.R

```diff
@@ -207,20 +207,31 @@ resetSearch <- function(pkgNamespaces) {
 getMappingPlottingData <- function(study = study, modelID = modelID, featureID = featureID, testID = testID, libraries = NULL) {
   mapping <- getMapping(study, libraries = libraries)
 
+  # stop if mapping object contains only one element and is not called "defaults"
```

Comment:
Conclusion from our Slack discussion: let's switch to consistently using `modelID = "default"`, and not make it a hard requirement that a single mapping table be named `"default"` (this is recommended, but it's hard to predict future use cases)

Reply:
Correcting message from 1st commit below, first bullet point: 
- study obj accepts multiple mappings. For one single element, it may be called whatever user wants it to be named (but in description of addMapping, it recommends being 'default'). Other mapping list elements must be named after modelIDs.